src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -231,6 +231,11 @@ export class OrganizeImportsAction extends EditorAction {
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyO,
 				weight: KeybindingWeight.EditorContrib
 			},
+			metadata: {
+
+				description: nls.localize2('organizeImports.description',
+					"Organize and optimize imports in the current file")
```

Comment:
Since  different languages implement this in different ways, I think this should be: 

```
Organize imports in the current file. Also called 'Optimize Imports' by some tools
```